test/child_key_warning_test.dart

```diff
@@ -13,7 +13,8 @@ int _nextFactoryId = 0;
 ///
 /// This prevents React JS from not printing key warnings it deems as "duplicates".
 void renderWithUniqueOwnerName(ReactElement render()) {
-  ReactDartComponentFactoryProxy factory = react.registerComponent(() => new _OwnerHelperComponent());
+  ReactDartComponentFactoryProxy factory =
```

Comment:
#nit I think in strong mode the LHS typing is unnecessary with this cast. Also, unnecessary parentheses? Could do:

```dart
final factory =
    react.registerComponent(() => new _OwnerHelperComponent()) as ReactDartComponentFactoryProxy.
```